prow/oss/cluster/monitoring/mixins/prometheus/prometheus.libsonnet

```diff
@@ -10,6 +10,4 @@
 (import 'stale_alerts.libsonnet') +
 (import 'tide_alerts.libsonnet') +
 (import 'prober_alerts.libsonnet') +
-(import 'boskos_alerts.libsonnet') +
-(import 'plank_alerts.libsonnet') +
```

Comment:
We don't want plank alerts?

Reply:
The plank alert in k8s prow is a heartbeat job, which doesn't exist in oss prow yet. Will add the plank alert once the heartbeat job is up.

k8s plank alert: https://github.com/kubernetes/test-infra/blob/master/config/prow/cluster/monitoring/mixins/prometheus/plank_alerts.libsonnet#L17